src/style.css

```diff
@@ -0,0 +1,35 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-left: 4px;
+  height: 16px;
+  width: 16px;
+}
+
+.stars--0 .stars__star {
+  background-image: url(/images/star.svg);
+}
```

Comment:
I don't think you need this one. In the selector above you are already doing that, so it does not seem to affect your stars behavior at all. Check this out and let me know